drawBot/context/baseContext.py

```diff
@@ -2143,8 +2322,8 @@ class BaseContext(object):
     _bezierPathClass = BezierPath
     _gradientClass = Gradient
 
-    fileExtensions = []
-    saveImageOptions = []
+    fileExtensions: list[str] = []
+    saveImageOptions: list[str] = []
```

Comment:
Code using this generally writes `for a, b in xxx.saveImageOptions`, and I guess the type annotation should be `list[tuple[str, str]]`

Reply:
fixed with b3bc4777f1b748bfbcccf8c0ffad462379f58fb8

Reply:
mmm, I reverted the change. The annotations do not align correctly.